builder/source/simple.go

```diff
@@ -112,50 +113,52 @@ func (s *SimpleSource) IsFetched() bool {
 	return PathExists(s.GetPath(s.validator))
 }
 
-// download utilises CURL to do all downloads
+// download downloads simple files using go grab
 func (s *SimpleSource) download(destination string) error {
-	hnd := curl.EasyInit()
-	defer hnd.Cleanup()
 
-	hnd.Setopt(curl.OPT_URL, s.URI)
-	hnd.Setopt(curl.OPT_FOLLOWLOCATION, 1)
+	file, err := os.Create(destination)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
 
-	out, err := os.Create(destination)
+	client := grab.NewClient()
+	req, err := grab.NewRequest(destination, s.URI)
 	if err != nil {
 		return err
 	}
 
-	pbar := pb.New64(0)
-	pbar.Set(pb.Bytes, true)
-	pbar.Set("prefix", filepath.Base(destination))
-	pbar.SetMaxWidth(80)
+	resp := client.Do(req)
```

Comment:
As `client` is not re-used, I would just do:

```suggestion
	resp := grab.NewClient().Do(req)
```